web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FrontPageController.java

```diff
@@ -3,19 +3,24 @@
 import edu.unc.lib.boxc.auth.api.models.AccessGroupSet;
 import edu.unc.lib.boxc.auth.fcrepo.services.GroupsThreadStore;
 import edu.unc.lib.boxc.web.common.controllers.AbstractErrorHandlingSearchController;
+import edu.unc.lib.boxc.web.common.utils.SerializationUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
 
 import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
 /**
  * Controller which populates the dynamic components making up the front page of
  * the public UI
- * @author bbpennel
+ * @author bbpennel, krwong
```

Comment:
`handleRequest` should be cleaned up to remove model attributes that aren't needed anymore. I'm sort of guessing none of it is needed aside from `return "frontPage";` but i haven't tried it